hw_1/main_1.py

```diff
@@ -0,0 +1,10 @@
+"""
+Найдите сумму цифр трехзначного числа.
+"""
+import math
+sum = int(input())
+a = math.floor(sum * 0.01)
+b = math.floor((sum - a * 100) * 0.1)
+c = sum - a * 100 - b * 10
+print(str(a) + " " + str(b) + " " + str(c))
```

Comment:
Если вы просто перечислите полученные результаты, вывод будет тем же.
```suggestion
print(a, b, c)
```